invenio_rdm_records/fixtures/vocabularies.py

```diff
@@ -106,84 +109,249 @@ def __iter__(self):
 
 
 #
-# Fixture
+# Exceptions
 #
-class VocabulariesFixture:
-    """Vocabularies fixture."""
+class ConflictingFixturesError(Exception):
+    """Exception when multiple modules provide same subject."""
 
-    def __init__(self, identity, search_paths, filename):
-        """Initialize the fixture."""
-        self._search_paths = search_paths
-        self._filename = filename
+    def __init__(self, errors):
+        """Constructor."""
+        message = "\n".join(errors)
+        super().__init__(message)
+        self.errors = errors
+
+
+#
+# Fixture
+#
```

Comment:
What about `MultiLocationVocabulariesFixture` and below `FileVocabulariesFixture`... then the interface inside can be sort of similar, being `load` the most important method to the users of this classes.

Reply:
How about:
 
`VocabulariesFixtures` -> `PrioritizedVocabulariesFixtures`

and keeping `VocabulariesFixture` the same?

Reply:
What about `PrioritizedVocabulariesFixture`, vocabularies is already plural 😊

Reply:
Discussed "IRL", stays as `PrioritizedVocabulariesFixtures`